apps/src/templates/codeReviewGroups/Student.jsx

```diff
@@ -41,24 +43,22 @@ export default function CodeReviewGroupMember({followerId, name, index}) {
   );
 }
 
-CodeReviewGroupMember.propTypes = {
+Student.propTypes = {
   followerId: PropTypes.number.isRequired,
   name: PropTypes.string.isRequired,
   index: PropTypes.number.isRequired
 };
 
-const getMemberStyle = (isDragging, draggableStyle) => ({
-  // some basic styles to make the group members look a bit nicer
+const getStudentStyle = (isDragging, draggableStyle) => ({
   userSelect: 'none',
   padding: grid * 2,
-  margin: grid,
-  borderRadius: '30px',
-  color: 'white',
+  color: color.dark_charcoal,
   width: 'auto',
   height: '20px',
+  border: `1px solid ${color.lighter_gray}`,
 
   // change background colour if dragging
-  background: isDragging ? 'navy' : '#0094CA',
+  background: isDragging ? 'navy' : 'white',
```

Comment:
nit: can use `color.white` here

Reply:
I think we'll probably update the navy to drag color that is a Code.org color at some point, going to punt this until then.